events/wc_druidic_events.txt

```diff
@@ -148,7 +205,112 @@ narrative_event = {
 		}
 	}
 }
-# Suggests liege use worgens against enemies
+# Arugal returns to Gilneas, or not...
+narrative_event = {
+	id = WCDRU.304
```

Comment:
This event can't fire since it's not in Warcraft-Guardians-of-Azeroth\common\on_actions\on_actions.txt
Plus, I'm afraid it will be CPU heavy. I recommend to fire it when the Plague happens (event WCPOD.2000) and scope to Arugal via `c_<Char ID>` scope.
https://ck2.paradoxwikis.com/Scopes#Character
https://ck2.paradoxwikis.com/Event_modding#Weight_Multiplier

Reply:
> Plus, I'm afraid it will be CPU heavy. I recommend to fire it when the Plague happens (event `WCPOD.2000`) and scope to Arugal via `c_<Char ID>` scope.

If the event can only fire once for a character with a unique flag, shouldn't it be fairly light CPU-wise?

If it needs to be tied to WCPOD.2000, I presume the code is inserted after line 92. A 365 day delay should allow enough time for the plague to have reached Dalaran before the event fires.

c_46055 = {
			narrative_event = { id = WCDRU.304 days = 365 }
		}

Reply:
> If the event can only fire once for a character with a unique flag, shouldn't it be fairly light CPU-wise?

Nope, the event tries to fire every year for every character. Only filtering pre-triggers can make it less heavy.
https://ck2.paradoxwikis.com/Event_modding#Filtering_pre-triggers

> If it needs to be tied to WCPOD.2000, I presume the code is inserted after line 92. A 365 day delay should allow enough time for the plague to have reached Dalaran before the event fires.

Nope, after line 91. It must be inside `if` block that checks `has_global_flag = plague_of_lordaeron_triggered_flag`. Otherwise, your event chain will happen after the first Plague outbreak, the Plague of Northrend, not after the Plague of Lordaeron.

Reply:
None of the filtering pre-triggers there are specific enough to reduce performance requirements, so I've implemented your suggestion. Should I remove the trigger requirements for WCDRU.304 now that WCPOD.2000 fires it?

Reply:
> Should I remove the trigger requirements for WCDRU.304 now that WCPOD.2000 fires it?

Yep, but there's another problem. You don't check Arugal's conditions when you fire WCDRU.304: religion, race trait and so on. I recommend to use `fury_of_wolf_human_event_trigger = yes`. It checks race trait, imprisonment, incapability, religion (not mad Old Gods' follower) and many more.